### PR TITLE
Development

### PR DESCRIPTION
**Pull request by onedebos**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (mosaaleb) [inline index.html:1]: `<!DOCTYPE html>` is not `html` tag, it tells the browser what version of `html` this page is written in. You still need the opening `html` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <script src="https://kit.fontawesome.com/e6d0becbe7.js"></script>
+</head>
+
+<body>
+    <main>
+        <section class="hero">
+            <section class="menu">
+                <ul class="menu-items">
+                    <li class="menu-item"><img src="assets/images/apple-icon.png" alt="apple-icon"></li>
+                    <li class="menu-item">Store</li>
```
Comment (mosaaleb) [inline index.html]: All the `li` here, needs to be clickable links.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                        
+                        <li class="menu-item"><input type="text" /><i class="fas fa-search"></i></li>
+
+            
+
+                </ul>
+            </section>
+            <img src="assets/images/hero-img1.png" alt="what will your verse be" class="hero-text-1" />
+
+            <img src="assets/images/hero-p2.png" alt="everyone has something to share." class="hero-text-2" />
```
Comment (mosaaleb) [inline index.html]: These text images are also clickable links, let's contain it inside a `div` to be more modular and add the links around them.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+            <img src="assets/images/hero-p3.png" alt="Watch the your verse film" class="hero-text-3" />
+
+        </section>
+
+        <section class="iphones">
+            <div class="iphone-5-orange">
```
Comment (mosaaleb) [inline index.html]: The images inside `iphones` section, are also links, let's add this.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            <div class="birthday-mac">
+                <img src="assets/images/birthdaymac_364x200.jpg" class="iphone-5s" alt="happy birthday mac" />
+            </div>
+            <div class="supplier-mac">
+                <img src="assets/images/supplier.jpg" class="supplier-mac" alt="iphone 5s" />
+            </div>
+
+        </section>
+    </main>
+
```
Comment (mosaaleb) [inline index.html:133]: There is no footer. Let's add the `<footer>` section.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/e6d0becbe7.js"></script>
+</head>
+
+<body>
+    <main>
+        <section class="hero">
+            <section class="menu">
```
Comment (mosaaleb) [inline index.html]: Let's replace `section` with `nav` to have semantic tags.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Hi @onedebos :wave:,
The UI looks very nice, Well done.

I have added some comments to handle first before merging. Also there is no footer, let's add it.

Let's repolish:
- The search icon needs to be vertically centered inside the form input.
- Also the links are not centered inside the navbar. Let's fix this.
Screenshot from 2019-07-14 03-17-04
- We need to use the [stickler](https://github.com/microverseinc/linters-config). It's very useful tool to make sure that our code has no errors, no typos, and our code styling guides are healthy.

Please make these changes and submit a new code review.

Feel free to contact me on slack if you have any questions.

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,118 @@
+*,::before,::after{
```
Comment (mosaaleb) [inline styles.css]: Very Nice.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+            <img src="assets/images/hero-p3.png" alt="Watch the your verse film" class="hero-text-3" />
+
+        </section>
+
+        <section class="iphones">
+            <div class="iphone-5-orange">
+                <img src="assets/images/iphone5s_364x200.jpg" class="iphone-5s" alt="iphone 5s" />
+            </div>
+            <div class="iphone-5-green">
```
Comment (mosaaleb) [inline index.html]: The four images supposed to have the same width and height, but they have different classes `iphone-5s` and `supplier-mac` with different sizing. Let's make the width and height unique.

Comment (mosaaleb) [reply index.html]: You can use `width: 25%` for every image, since you have four images.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+.menu-items input{
+    border-radius:5px;
+    background:linear-gradient(#1E1E1F, #726B6F);
+    border:1px solid #333;
+    padding-left: 10px;
+    color:white;
+    margin-right:10px;
+}
+
+i{
```
Comment (mosaaleb) [inline styles.css]: typo?

**Review by mosaaleb — CHANGES_REQUESTED** [review]

File: styles.css (hunk 9 above)
Comment (onedebos) [reply styles.css]: This was to style my font-awesome icon.

File: index.html (hunk 8 above)
Comment (onedebos) [reply index.html]: I think the supplier-mac image is a bit wider than the rest.

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+.menu-items li{
+    list-style: none;
+    font-weight: 500;
+    color:#fff;
+    margin: 2px 0;
+}
+
+.menu-items a{
```
Comment (mosaaleb) [inline styles.css]: For a better user experience, We need to add hover style to the nav links

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Hi @onedebos 👋,

#### Nice work on the footer section :clap:

Let's make sure requirements are all done:

- [x] The nav-bar is still unfixed, let's center the links inside the navbar, and also fix the search icon position.

- [ ] We need to work on the Indentation style, and the overall code slyling guides. A way to ensure that you have healthy code base, is by using linters locally (as extension in your preferred code editor), or when you push your code to the repo using [stickler](https://github.com/microverseinc/linters-config). Feel free to DM me on slack if you don't know how to do this.

Let's make sure that all requirments are done. Then please submit a new code review. 
### Happy coding :man_technologist:

**Comment by onedebos** [comment]
I have just included CSS and media-queries to try to get all my styles to show up the same in different browsers and different screen sizes. I use a 14' laptop screen and so some times my styling looks a bit off on other computers.

**Comment by mosaaleb** [comment]
### Hi @onedebos :wave:,
Well done. Looks good to me. :1st_place_medal: 
You can merge into master now. Don't forget to submit a project completion.

**Review by mosaaleb — APPROVED** [review]

**Pull request merged**